docs/docs/overview/quickstart/get-started-with-java.md

```diff
@@ -0,0 +1,358 @@
+            triggerEventRequest.setName("name");
+            triggerEventRequest.setTo(subscriberRequest);
+            triggerEventRequest.setPayload(Collections.singletonMap("customVariables", "Hello"));
+
+
+    try {
+        return novu.triggerEvent(triggerEventRequest);
+    }catch (Exception e){
+        log.error("Error triggering event", e);
+    }
+
+            return null;
+
+```
+
+Before running the code, make sure you understand the following:
+
+- The value of `payload` is an array of the data that you want to be dynamically injected into the notification workflow content.
+- The value of `subscriberId` is the ID of the subscriber on Novu. Replace `12345` with your subscriber ID.
+
+Run the code to trigger a notification! :e-mail:
+
+## Topics
+
+Novu provides a simple API that offers an easy interface for triggering notifications to multiple subscribers at once. This API is called **Topics** and allows users to manage their bulk notifications without having to implement complex loops. A topic is identified by a custom key that is provided by the user, and this key will be the identifier used in the Topics API.
+
+::::info
+The topic key should be unique and can't be changed once chosen. Novu also caters for key uniqueness behind the scenes.
+::::
+
+A topic can have multiple subscribers who will receive a notification whenever a message is sent to the topic.
+
+### Create a Topic
+
+Copy and paste the following code into your app to create a topic:
+
+```java
+String apiKey = "API_KEY";
+Novu novu = new Novu(apiKey);
+
+TopicRequest createTopicRequest = new TopicRequest();
+    createTopicRequest.setKey("key");
+    createTopicRequest.setName("name");
+        try {
+            return novu.createTopic(createTopicRequest);
+        }catch (Exception e){
+            log.error("Error creating Topic", e);
+        }
+
+        return null;
+```
+
+Before running the code, make sure you understand the following:
+
+- When creating a `key`, ensure it is unique and accurately identifies the topic. Document naming conventions and communicate them to team members to avoid confusion and ensure a smooth workflow.
+- The value of `name` should be a descriptive topic name.
+
+### Add subscribers to a Topic
+
+Copy and paste the following code into your app to add subscribers a topic:
+
+```java
+String apiKey = "API_KEY";
+Novu novu = new Novu(apiKey);
+
+    SubscriberRequest subscriberRequest = new SubscriberRequest();
+        subscriberRequest.setEmail("email");
+        subscriberRequest.setFirstName("fName");
+        subscriberRequest.setLastName("lName");
+        subscriberRequest.setPhone("phone");
+        subscriberRequest.setAvatar("avatar");
+        subscriberRequest.setSubscriberId("subscriberId");
+
+        String topicKey = "key";
+        SubscriberAdditionRequest requestBody = new SubscriberAdditionRequest();
+        requestBody.setSubscribers(Collections.singletonList(subscriberRequest));
```

Comment:
This should be a list of Strings not list of Objects. See https://github.com/novuhq/novu-java/pull/30